src/main/resources/arguments.json

```diff
@@ -510,6 +510,23 @@
       }
     }
   },
+  "remindme*": {
+    "delay*": {
+      "_type": "required",
+      "message": {
+        "_type": "required"
+      }
+    },
+    "dm*": {
+      "delay*": {
+        "_type": "required",
+        "message": {
+          "_type": "required"
+        }
+      }
+    },
+    "list": {}
```

Comment:
this should have a type of `command`

Reply:
Doesn't technically need it, I tend to leave it blank